src/main/java/net/minecraftforge/common/capabilities/Capability.java

```diff
@@ -65,10 +65,29 @@
      * the fully qualified class name for the target interface.
      */
     public String getName() { return name; }
+
     /**
      * @return An instance of the default storage handler. You can safely use this store your default implementation in NBT.
      */
     public IStorage<T> getStorage() { return storage; }
+    
+    /**
+     * Quick access to the IStorage's readNBT. 
+     * See {@link IStorage#readNBT(Capability, Object, EnumFacing, NBTBase)}  for documentation.
+     */
+    public void readNBT(T instance, EnumFacing side, NBTBase nbt)
```

Comment:
Seeing as it's just a forwarding call, it makes sense to name it the same as what you're forwarding to